script/Arcueid/Day 11/11_00_ARC11_2.txt

```diff
@@ -404,8 +404,9 @@ I mean of course the vampiric impulse―――an innate desire, a %{i}craving%{/
 [sha:71a679bfce54efc4cd68983c9fe3d4028dc00bb8]{
 -- Page 64, Offset 29683.
 -- 原因は生命体としてのスケールの違いとも、彼らが血液を魂として捉えている、とも言われています」
-Who can say whether it's down to the sheer difference in ability between us as species,
- or whether by sucking someone's blood they steal their very soul."
+// Rephrased to avoid using "who can say" again
```

Comment:
Remember to delete comments that only make sense in the context of the PR before merging